components/CategoryList.tsx

```diff
@@ -88,9 +88,14 @@ export const CategoryList: React.ElementType = ({ itemStyle }: any) => {
     },
     {
       type: LinkType.LINK,
-      name: "Podcasts",
+      name: "Podcast",
       url: "/podcasts/",
     } as LinkLink,
+    {
+      type: LinkType.LINK,
+      name: "Video",
+      url: "https://www.youtube.com/channel/UCWg3QqUzqxXt6herm5sMjNw",
```

Comment:
Can you refactor this podcasts url so that is just defined once in a `constants.ts` file? That way it will be easier to change later -- and we can use this constants file for other static links like social media, etc.

Reply:
Done